pkg/azureconstants/azure_constants.go

```diff
@@ -58,11 +58,15 @@ const (
 	PerfProfileNone               = "none"
 	PremiumAccountPrefix          = "premium"
 	PvcNameKey                    = "csi.storage.k8s.io/pvc/name"
+	PvcNameLabel                  = "csi.storage.k8s.io/pvc"
 	PvcNamespaceKey               = "csi.storage.k8s.io/pvc/namespace"
 	PvcNamespaceTag               = "kubernetes.io-created-for-pvc-namespace"
+	PvcNamespaceLabel             = "csi.storage.k8s.io/pvc-namespace"
 	PvcNameTag                    = "kubernetes.io-created-for-pvc-name"
 	PvNameKey                     = "csi.storage.k8s.io/pv/name"
 	PvNameTag                     = "kubernetes.io-created-for-pv-name"
+	PvNameLabel                   = "csi.storage.k8s.io/pv"
+	VolumeAttachmentKey           = "csi.storage.k8s.io/volumeattachment/name"
```

Comment:
Is this defined by CSI? If not, we should use our own annotation namespace, e.g. `disk.csi.azure.com/volumeattachment/name`.